docs/prem-app/installation/install-linux-production.md

```diff
@@ -42,6 +42,4 @@ cd ./prem-app
 docker-compose up -f docker-compose.yml -f docker-compose.gpu.yml -d
 ```
 
-And you will have the UI at `http://{localhost|server_ip}:1420`. 
-
-> Make sure that in `Settings` the Backend URL is set to `http://{localhost|server_ip}:8000`.
\ No newline at end of file
+And you will have the UI at `http://{localhost|server_ip}:8000`.
```

Comment:
Are you sure? this only happens with Caddy in front, not bare docker compose

Reply:
Now caddy is available in prem-app/docker-compose.yml as well.